examples/controls/control.css

```diff
@@ -0,0 +1,12 @@
+#controlDiv {
+      position:absolute;
+      top:10px;
+      right:10px;
+      width:150px;
+      height:150px;
+      background-color: white;
+      border-style: solid;
+      border-color: black;
+      border-width: 1px;
+      box-shadow: 1px 1px 10px #555;
```

Comment:
(for border and boxes, some people prefer staying with px, they think it's nicer)